packages/neuron-ui/src/widgets/MnemonicInput/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import styles from './index.module.scss'
+
+const MnemonicInput = ({
+  disabled,
+  words,
+  inputsWords,
+  onChangeInputWord,
+}: {
+  disabled: boolean
+  words: string
+  inputsWords: string[]
+  onChangeInputWord: React.ChangeEventHandler<HTMLInputElement>
+}) => {
+  const wordList = useMemo(() => (words ? words.split(' ') : new Array(12).fill('')), [words])
```

Comment:
better check if the length of `words.split(' ')` is 12, and this logic be brief as `words?.split(' ') ?? new Array(12).fill('')`